client/src/screens/group.screen.js

```diff
@@ -206,8 +207,6 @@ class Group extends Component {
 
   leaveGroup() {
     this.props.leaveGroupQry({ groupId: this.getGroup().id }).then(() => {
-      // pull group details down again to populate the group details
-      this.onRefresh();
```

Comment:
this seems like an unrelated change?

Reply:
It's semi related as it's a totally useless refresh that was working while the other failed.

Reply:
👍